main.tf

```diff
@@ -364,7 +364,7 @@ data "aws_iam_policy_document" "ecs_tasks" {
 
     principals {
       type        = "Service"
-      identifiers = ["ecs-tasks.amazonaws.com"]
+      identifiers = concat(["ecs-tasks.amazonaws.com"], var.trusted_principals)
```

Comment:
It is better to have `compact(distinct(concat(...)))` to remove empty and repetitive values. Could you please update this?

Reply:
Done 👍